.lintstagedrc.json

```
@@ -1,7 +1,8 @@
 {
-  "*": "prettier --ignore-unknown --write",
+  "!*.md": "prettier --ignore-unknown --write",
   "*.md": [
     "markdownlint-cli2-fix",
+    "prettier --write",
     "node scripts/front-matter_linter.js --fix true"
```

```suggestion
    "markdownlint-cli2 --fix",
    "node scripts/front-matter_linter.js --fix true",
    "prettier --write"
```

a. The `markdownlint-cli2-fix` [is deprecated](https://github.com/DavidAnson/markdownlint-cli2#command-line). Use `--fix` option instead. We need to update package.json as well.
b. Recently we resolved clashes between Prettier and front-matter linter. For future proofing we always run Prettier last.